src/components/account/account.test.tsx

```diff
@@ -93,13 +94,14 @@ describe('account test suite', () => {
 
         const response = await account.getGoogleOIDCCallback(ctx, params);
         expect(response.body).toContain('Access Denied');
+        expect(testSpacing(response.body as string)).toHaveLength(0);
```

Comment:
These two lines seem incompatible. How can `response.body` include *Access Denied* and yet be empty?

Reply:
This response is wrapped in the `testSpacing` function, which returns an array list. If the list is not empty, it should throw an error.

It's purpose is to check the HTML content and establish if cases[with no spacing]()occur for the end user.